.github/workflows/test.yml

```diff
@@ -11,8 +11,6 @@ jobs:
         uses: actions/checkout@v3
       - name: Step 2 - Getting scarb
         uses: software-mansion/setup-scarb@v1
```

Comment:
Could prob update this too to use latest `uses: software-mansion/setup-scarb@v1.3.2`

Reply:
Referring to the scarb action version here 😄

Reply:
ah, ok